docs/csharp/programming-guide/concepts/async/snippets/async-return-types/AsyncStreams.cs

```diff
@@ -5,38 +5,38 @@
 
 namespace AsyncExamples
 {
-
     public static class AsyncStreamExample
     {
-
-        public static async Task Examples()
+        public static async Task ReadWordsAsync()
         {
-            await foreach (var word in ReadWordsFromStream())
+            await foreach (string? word in ReadWordsFromStreamAsync())
```

Comment:
Why this is nullable string while the `ReadWordsFromStreamAsync` returns `IAsyncEnumerable<string>`?